app/src/main/java/com/example/topmovies/view/CircleAvatarView.kt

```diff
@@ -54,7 +54,7 @@ class CircleAvatarView @JvmOverloads constructor(
                 label = typedArray.getString(R.styleable.CircleAvatarView_text) ?: ""
                 imageBitmap = BitmapFactory.decodeResource(
                     resources, typedArray.getResourceId(
-                        R.styleable.CircleAvatarView_src, 0
+                        R.styleable.CircleAvatarView_src, R.drawable.def_profile_image
```

Comment:
That wasn't part of your assignment.  Even if you are doing non-related work, that should be a separate commit.